CP04OSSM/CP04OSSM_R00008_ingest.csv

```diff
@@ -1,43 +1,43 @@
-mi.dataset.driver.cg_dcl_eng.dcl.cg_dcl_eng_dcl_recovered_driver,/omc_data/whoi/OMC/CP04OSSM/R00008/cg_data/dcl12/syslog/*.syslog.log,CP04OSSM-SBD12-00-DCLENG000,recovered_host,Expected,
-mi.dataset.driver.hyd_o.dcl.hyd_o_dcl_recovered_driver,/omc_data/whoi/OMC/CP04OSSM/R00008/cg_data/dcl12/hyd2/*.hyd2.log,CP04OSSM-SBD12-03-HYDGN0000,recovered_host,Expected,
-mi.dataset.driver.pco2a_a.dcl.pco2a_a_dcl_recovered_driver,/omc_data/whoi/OMC/CP04OSSM/R00008/cg_data/dcl12/pco2a/*.pco2a.log,CP04OSSM-SBD12-04-PCO2AA000,recovered_host,Expected,
-mi.dataset.driver.zplsc_c.zplsc_c_recovered_driver,/omc_data/whoi/OMC/CP04OSSM/R00008/instruments/dcl37/ZPLSC*/DATA/*/*.01A,CP04OSSM-MFD37-07-ZPLSCC000,recovered_inst,Expected,
\ No newline at end of file
+mi.dataset.driver.ctdbp_cdef.dcl.ctdbp_cdef_dcl_recovered_driver,/omc_data/whoi/OMC/CP04OSSM/R00008/cg_data/dcl37/ctdbp2/*.ctdbp2.log,CP04OSSM-MFD37-03-CTDBPE000,recovered_host,Available,
+mi.dataset.driver.ctdbp_cdef.ctdbp_cdef_recovered_driver,/omc_data/whoi/OMC/CP04OSSM/R00008/instruments/dcl37/CTDBP_sn*/SBE*.hex,CP04OSSM-MFD37-03-CTDBPE000,recovered_inst,Available,
+mi.dataset.driver.dosta_abcdjm.dcl.dosta_abcdjm_dcl_recovered_driver,/omc_data/whoi/OMC/CP04OSSM/R00008/cg_data/dcl37/dosta2/*dosta2.log,CP04OSSM-MFD37-04-DOSTAD000,recovered_host,Available,
+mi.dataset.driver.zplsc_c.dcl.zplsc_c_dcl_telemetered_driver,/omc_data/whoi/OMC/CP04OSSM/R00008/cg_data/dcl37/zplsc/*.zplsc.log,CP04OSSM-MFD37-07-ZPLSCC000,recovered_host,Available,
+mi.dataset.driver.cg_cpm_eng.cpm.cg_cpm_eng_cpm_recovered_driver,/omc_data/whoi/OMC/CP04OSSM/R00008/cg_data/cpm2/syslog/cpm_status*.txt,CP04OSSM-RIC21-00-CPMENG000,recovered_host,Available,
+mi.dataset.driver.cg_dcl_eng.dcl.cg_dcl_eng_dcl_recovered_driver,/omc_data/whoi/OMC/CP04OSSM/R00008/cg_data/dcl26/syslog/*.syslog.log,CP04OSSM-RID26-00-DCLENG000,recovered_host,Available,
+mi.dataset.driver.velpt_ab.dcl.velpt_ab_dcl_recovered_driver,/omc_data/whoi/OMC/CP04OSSM/R00008/cg_data/dcl26/velpt1/*.velpt1.log,CP04OSSM-RID26-04-VELPTA000,recovered_host,Available,
+mi.dataset.driver.velpt_ab.velpt_ab_recovered_driver,/omc_data/whoi/OMC/CP04OSSM/R00008/instruments/dcl26/VELPT_snAQD*/*.aqd,CP04OSSM-RID26-04-VELPTA000,recovered_inst,Available,
+mi.dataset.driver.phsen_abcdef.dcl.phsen_abcdef_dcl_recovered_driver,/omc_data/whoi/OMC/CP04OSSM/R00008/cg_data/dcl26/phsen1/*.phsen1.log,CP04OSSM-RID26-06-PHSEND000,recovered_host,Available,
+mi.dataset.driver.phsen_abcdef.phsen_abcdef_recovered_driver,/omc_data/whoi/OMC/CP04OSSM/R00008/instruments/dcl26/PHSEN_snP*/SAMI*.txt,CP04OSSM-RID26-06-PHSEND000,recovered_inst,Available,
+mi.dataset.driver.suna.suna_dcl_driver,/omc_data/whoi/OMC/CP04OSSM/R00008/cg_data/dcl26/nutnr/*.nutnr.log,CP04OSSM-RID26-07-NUTNRB000,recovered_host,Available,
+mi.dataset.driver.suna.suna_instrument_driver,/omc_data/whoi/OMC/CP04OSSM/R00008/instruments/dcl26/NUTNR_sn*/DAT/*.CSV,CP04OSSM-RID26-07-NUTNRB000,recovered_inst,Available,

```

Comment:
SUNA was deployed, need correct SUNA driver/parser and filemask

Reply:
Confirmed a SUNA was deployed.